pallets/registrar/src/lib.rs

```diff
@@ -244,6 +273,100 @@ pub mod pallet {
         NotSufficientDeposit,
     }
 
+    #[pallet::hooks]
+    impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+        #[cfg(feature = "std")]
+        fn integrity_test() {
```

Comment:
```suggestion
        fn try_state(_block_number: u32) {
```

The `integrity_test` hook doesn't have access to the storage, despite docs saying otherwise.